x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/form.tsx

```diff
@@ -298,12 +299,15 @@ export function useOnSubmit({
         }
       }
 
+      const agentPolicyIdToSave = createdPolicy?.id ?? packagePolicy.policy_id;
+      const forceInstall = force || agentPolicyIdToSave === AGENTLESS_POLICY_ID;
```

Comment:
maybe worth checking we are in serverless too

Reply:
potentially there will be agentless on ESS, the work kicked off in parallel, but I agree that it makes sense to be on the safer side for now and check for serverless, or at least that it's not on-prem. will look into it

Reply:
@nchaulet updated the pr including the check for serverless env and agentless FF being enabled